lib/opensusebasetest.pm

```diff
@@ -306,6 +306,10 @@ sub wait_boot {
         if (get_var('ONLINE_MIGRATION')) {
             push @tags, 'migration-source-system-grub2';
         }
+
+        # enter password before grub when FULL_LVM_ENCRYPT=1
+        workaround_type_encrypted_passphrase;
```

Comment:
your comment to function is confusing. If you enter inside a function you will find out that to actually call unlock_if_encrypted there are THREE possible cases :
1.  ENCRYPT= 1 AND FULL_LVM_ENCRYPT=1
2. ARCH= ppc64le AND ENCRYPT=1 AND ENCRYPT_ACTIVATE_EXISTING=0
3. ENCRYPT_FORCE_RECOMPUTE=1 AND ENCRYPT=1

so stating   just "when FULL_LVM_ENCRYPT=1" at least ENCRYPT=1 is worth to mention as it used in all 3 cases